test/functional/feature_dbcrash.py

```diff
@@ -257,13 +255,14 @@ def run_test(self):
                     self.nodes[3],
                     nblocks=min(10, current_height + 1 - self.nodes[3].getblockcount()),
                     # new address to avoid mining a block that has just been invalidated
-                    address=self.nodes[3].getnewaddress(),
+                    address=getnewdestination()[2],
                     sync_fun=self.no_op,
                 ))
             self.log.debug(f"Syncing {len(block_hashes)} new blocks...")
             self.sync_node3blocks(block_hashes)
-            utxo_list = self.nodes[3].listunspent()
-            self.log.debug(f"Node3 utxo count: {len(utxo_list)}")
+            self.wallet.rescan_utxos()
+            utxo_list = self.wallet._utxos
```

Comment:
Not sure about using the private member here. What about `    def get_utxos(self, *, mark_as_spent=True): return copy.copy(self._utxos)`  or so?

Reply:
Agree, accessing private members should be avoided. Thanks, done.